commands.go

```diff
@@ -26,19 +26,16 @@ var commandsTree = newTrie()
 
 // Collection of minio flags currently supported.
 var globalFlags = []cli.Flag{
-	cli.BoolFlag{
-		Name:  "quiet, q",
-		Usage: "Suppress chatty console output.",
-	},
-	cli.BoolFlag{
-		Name:  "debug",
-		Usage: "Enable debugging output.",
-	},
 	cli.StringFlag{
 		Name:  "config-dir, C",
 		Value: mustGetConfigPath(),
 		Usage: "Path to configuration folder.",
 	},
+	cli.StringFlag{
+		Name:  "profile-mode",
+		Value: "",
+		Usage: "Enable profiling mode, one of [cpu, mem, block].",
+	},
```

Comment:
we should do this via environment variable.
MINIO_PROFILER=off | cpu | mem | block